modules/ROOT/pages/oas3-support.adoc

```diff
@@ -0,0 +1,72 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer.
```

Comment:
Phase 1 of the support for OAS 3.0 enables consumption scenarios in the Anypoint platform. It allows users to showcase their existing external APIs in the platform to drive partner and customer adoption. It includes support for products like Anyppint Exchange, API Community Manager, and API Designer.

Reply:
@harshakirank , do you really want to use "consumption scenarios" in this doc? I have no idea what that means, and I doubt that readers will, either.

Reply:
@harshakirank , "Support for products like"? Don't you mean "support for Anypoint Exchange, API Community Manager, and API Designer"?

Reply:
@harshakirank , "In the platform"? What about ACM users who are showcasing their APIs in their own community sites?

Reply:
I see your point, I was trying to represent a use case as opposed to the products mentioned above. Maybe we should add use cases something like
With the support in the above products:
Users can catalog their existing OAS3 specs in the platform to enable partner and customer collaboration scenarios. Additionally, users can do edits to their specs with the API Designer.